### PR TITLE
re-adds being able to ghost out of posibrains / mmis

### PR DESCRIPTION
**Pull request by ShadowDeath6**
top text

:cl: ShadowDeath6
rscadd: You can now ghost out of posibrains and MMIs.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/brain/posibrain.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@ var/global/posibrain_notif_cooldown = 0
 	Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm.</b>"
 	var/new_mob_message = "<span class='notice'>The positronic brain chimes quietly.</span>"
 	var/dead_message = "<span class='deadsay'>It appears to be completely inactive. The reset light is blinking.</span>"
-	var/list/fluff_names = list("PBU","HIU","SINA","ARMA","OSI","HBL","MSO","RR","CHRI","CDB","HG","XSI","ORNG","GUN","KOR","MET","FRE","XIS","SLI","PKP","HOG","RZH","GOOF","MRPR","JJR","FIRC","INC","PHL","BGB","ANTR","MIW","WJ","JRD","CHOC","ANCL","JLLO","JNLG","KOS","TKRG","XAL","STLP","CBOS","DUNC","FXMC","DRSD")
+	var/list/fluff_names = list("PBU","HIU","SINA","ARMA","OSI","HBL","MSO","RR","CHRI","CDB","HG","XSI","ORNG","GUN","KOR","MET","FRE","XIS","SLI","PKP","HOG","RZH","GOOF","MRPR","JJR","FIRC","INC","PHL","BGB","ANTR","MIW","WJ","JRD","CHOC","ANCL","JLLO","JNLG","KOS","TKRG","XAL","STLP","CBOS","DUNC","FXMC","DRSD","CRX","SPR","MAC","EPH","OSN","XTD")
```
Comment (Time-Green) [inline code/modules/mob/living/carbon/brain/posibrain.dm:27]: You forgot TMGR

Comment (ShadowDeath6) [reply code/modules/mob/living/carbon/brain/posibrain.dm:27]: this diff doesn't exist, try again

Comment (Time-Green) [reply code/modules/mob/living/carbon/brain/posibrain.dm:27]: 7

**Comment by Mrcreepssss** [comment]
YUS! does the posi remain claimed tho?

**Comment by ShadowDeath6** [comment]
If you leave it someone else can enter

**Comment by Mrcreepssss** [comment]
makes sense to me!
though do you then get back into the posibrain when you click re-enter corpse?
what if you click it and someone else went in?
does the new person in get a new posibrain name?

**Comment by ShadowDeath6** [comment]
You can re enter it as long as nobody else has taken it
As far as names go idk

**Pull request merged**